src/components/InputsBox.jsx

```diff
@@ -1,36 +1,62 @@
-import { Button, Grid, TextField } from '@mui/material';
-import TextareaAutosize from '@mui/material/TextareaAutosize';
+import { Button, Grid, TextField, Box, Alert } from '@mui/material';
+import { useState } from 'react';
 
 export default function InputsBox() {
+    const [imageUrl, setImageUrl] = useState(null)
+    const [alert, setAlert] = useState(false)
+
+  function onImageChange(file) {
+    if(file && file.type.split('/')[0] === 'image'){
```

Comment:
Use optional chaining -> `file?.type.split('/')[0] === 'image'` no need for `file &&`